docs/docs/integrations/platforms/google.mdx

```diff
@@ -2,41 +2,56 @@
 
 All functionality related to [Google Cloud Platform](https://cloud.google.com/) and other `Google` products.
 
-## LLMs
-
-### Vertex AI
+## Chat models
 
-Access `PaLM` LLMs like `text-bison` and `code-bison` via `Google Vertex AI`.
+### ChatGoogleGenerativeAI
 
-We need to install `google-cloud-aiplatform` python package.
+Access `Gemini` models such as `gemini-pro` and `gemini-pro-vision` through the `ChatGoogleGenerativeAI` class.
 
 ```bash
-pip install google-cloud-aiplatform
+pip install -U langchain-google-genai
```

Comment:
@hinthornw would you mind adding the `url` parameter to the package metadata?

That way https://pypi.org/project/langchain-google-genai/ can link back to either LangChain repo root or the package's subfolder

Reply:
Will do! In a separate PR though

Reply:
Thanks! Sorry for lurking, wasn't sure a better way to communicate this to you

Reply:
Always appreciate the comments and requests!